packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -26,6 +26,8 @@ exports.onPreBootstrap = (gatsbyApi, themeOptions) => {
     'src/content',
     'src/content/files',
     'src/releases',
+    'static',
+    'static/downloads',
```

Comment:
I think it makes sense to bootstrap those folders, so it's clear what the convention is